### ISSUE TITLE
Low accuracy with case 2 (multiclass) notebook

### ISSUE THREAD

**Issue by lalondma** [qubilab/CNN-for-skin-lesion-classification#1]
Hi, I'm playing with your 2nd notebook on Colab (great stuff!) but I haven't been able to get good fine-tuning results: validation accuracy lingers around 20%. I tested the pretrained network but again, accuracy is about 28% where most of the test images are classified as VASC. I suspect an issue with image normalization. Any hint?
